docs/content/third-party-modules/opentracing.md

```diff
@@ -20,7 +20,7 @@ This document explains how to use OpenTracing with the Ingress Controller.
 1. **Use an Ingress Controller image that contains OpenTracing.** 
    - You can find the images that include OpenTracing listed [in the technical specs doc]({{< relref "technical-specifications.md#supported-docker-images" >}}). 
    - Alternatively, you can [build your own image]({{< relref "installation/building-ingress-controller-image.md" >}}) using `debian-image` (or `alpine-image`) for NGINX or `debian-image-plus` (or `alpine-image-plus`) for NGINX Plus.
-[Jaeger](https://github.com/jaegertracing/jaeger-client-cpp), [Zipkin](https://github.com/rnburn/zipkin-cpp-opentracing) and [Datadog](https://github.com/DataDog/dd-opentracing-cpp/) tracers are installed by default.
+[Jaeger](https://github.com/jaegertracing/jaeger-client-cpp),[New Relic](https://github.com/newrelic/nri-nginx), [Zipkin](https://github.com/rnburn/zipkin-cpp-opentracing) and [Datadog](https://github.com/DataDog/dd-opentracing-cpp/) tracers are installed by default.
```

Comment:
The statement in the document mentions libraries that we build into the project by default.  
AFAIK the project does not include the NewRelic agent.  And thus this edit would be a false statement.

That said, the project is going to move to OpenTelemetry.  Does the NewRelic platform support OpenTelemetry?
If it does, it would be useful to submit instructions around setting that up once we get there in the near future.